airflow/providers/amazon/aws/hooks/glue_crawler.py

```diff
@@ -105,7 +105,8 @@ def create_crawler(self, **crawler_kwargs) -> str:
         """
         crawler_name = crawler_kwargs['Name']
         self.log.info("Creating crawler: %s", crawler_name)
-        return self.glue_client.create_crawler(**crawler_kwargs)['Crawler']['Name']
```

Comment:
This is wrong and would lead to ```KeyError```

```
  File "/usr/local/lib/python3.8/site-packages/airflow/providers/amazon/aws/hooks/glue_crawler.py", line 108, in create_crawler
    return self.glue_client.create_crawler(**crawler_kwargs)['Crawler']['Name']
KeyError: 'Crawler'
```
According to : https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue.html#Glue.Client.create_crawler 

This returns an empty dict